### ISSUE TITLE
Can LBFGSB used in online learning? 

### ISSUE THREAD

**Issue by bizso09** [BRML/climin#25]
Can this implementation of L-BFGS-B used as a replacement for stochastic gradient descent?

**Comment by bayerj** [comment]
Typically, LBFGS has the assumption that the objective is deterministic and
not stochastic.

That does not keep you from trying it out though. I have used LBFGS with
neural nets and *really* large batch sizes, e.g. a 1000.

PS: Questions like these are better posted to the mailinglist.

On 15 January 2015 at 16:15, bizso09 <notifications@github.com> wrote:

> Can this implementation of L-BFGS-B used as a replacement for stochastic
> gradient descent?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/BRML/climin/issues/25>.
>



-- 
Q: Why is this email five sentences or less?
A: http://five.sentenc.es


Dipl. Inf. Justin Bayer
Lehrstuhl für Robotik und Echtzeitsysteme, Technische Universität München
http://www6.in.tum.de/Main/Bayerj
